segmenters/image/FaceNetSegmenter/requirements.txt

```diff
@@ -1,3 +1,3 @@
 facenet-pytorch==2.5.2
 torchvision==0.9.0
-opencv-python==4.5.1
\ No newline at end of file
+opencv-python==4.5.1.48
```

Comment:
lets keep just minor version, no need to set a patch version

Reply:
it somehow fails if I don't put in the last part

Reply:
what is somehow? can u point out why?

Reply:
yeah with ERROR: No matching distribution found for opencv-python==4.5.1

Reply:
and a 4.5.0?

Reply:
same error

Reply:
try in Dockerfile updating pip version before running the pip install -r ...

Reply:
WARNING: Value for scheme.headers does not match. Please report this to <
https://github.com/pypa/pip/issues/9617> followed with the same error, didn't find obvious solution online either

Reply:
use the .48 then